apps/framework-internal-app/project.toml

```diff
@@ -26,4 +26,4 @@ host_port = 3001
 "0.0" = "1fe7b86"
 "0.1" = "005108e"
 "0.2" = "969a0b7"
-"0.3" = "f8ebebb"
+"0.3" = "22d9fcb"
```

Comment:
Do we need to add 'description' back to UserActivity model as well?

Reply:
ah yes